README.md

```diff
@@ -66,6 +66,26 @@ When you use `Telescope neorg find_aof_project_tasks` you can pick an area of fo
 <video alt="find_aof_project_tasks" src="https://user-images.githubusercontent.com/81827001/158401841-9ca3a311-bac1-4733-9a6e-6125003d8a38.mov"></video>
 </details>
 
+Until the new GTD infrastructure arrives, you could collect all your todo items with a regex, for example:
+
+```lua
+-- Make sure you have ripgrep installed.
+-- Add the following function to your `~/.config/nvim/init.lua`:
+do
+    local _, neorg = pcall(require, "neorg.core")
+    local dirman = neorg.modules.get_module("core.dirman")
+    local current_workspace = dirman.get_current_workspace()
```

Comment:
needs to be inside the function
otherwise it won't work if you switch workspaces